src/Zigzag_persistence/doc/Intro_zigzag_persistence.h

```diff
@@ -0,0 +1,69 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Hannah Schreiber
+ *
+ *    Copyright (C) 2023 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#ifndef DOC_ZIGZAG_PERSISTENCE_INTRO_ZIGZAG_PERSISTENCE_H_
+#define DOC_ZIGZAG_PERSISTENCE_INTRO_ZIGZAG_PERSISTENCE_H_
+
+// needs namespace for Doxygen to link on classes
+namespace Gudhi {
+namespace zigzag_persistence {
+
+/** \defgroup zigzag_persistence Zigzag Persistence
+ * @{
+ * \author    Cl&eacute;ment Maria, Hannah Schreiber
+ *
+ * We refer to the introduction page \ref persistent_cohomology for persistent (co)homology for an introduction
+ * to the topic.
+ * Zigzag persistence is a generalization of the latter. While standard persistence only allows to grow the filtered
+ * complex by adding simplices, zigzag persistence also allows removals. Hence the name "zigzag", as the module
```

Comment:
Is the zigzag code restricted to simplicial complexes, or is it a bit more general (like usual persistence handles cubical complexes just fine, even if we keep using the confusing name Simplex_handle)?

Reply:
It is not restricted to simplicial complexes, just as the cohomology algorithm. So I replaced "simplices" by "faces".